### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by shreyas-9601**

### CODE REVIEW

File: order_management.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+from flask import *
+app = Flask(__name__)
+from orders_model import OrdersModel
+from order_validation import Order
+
+@app.route('/order', methods=['POST'])
+def create_order():
+    order_data = request.json
+    orders=Order(order_id=id, name=order_data['Name'], birthday = order_data['Birthday'],
```
Comment (shreyas-9601) [inline order_management.py]: removal of order_id

File: order_management.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    order_data = request.json
+    orders=Order(order_id=id, name=order_data['Name'], birthday = order_data['Birthday'], 
+                        email = order_data['Email'], state = order_data['State'], zipcode = order_data['ZipCode'])
+    order = OrdersModel(name = orders.user.name, birthday = orders.user.birthday, 
+                        email = orders.user.email, state = orders.user.state, zipcode = orders.user.zipcode)
+    if orders.validate_orders():
+        order = order.save()
+        return jsonify({"message":"Order created successfully", "order_id": str(order.id)}),201
+    else:
+        if(orders.user.is_valid_mail() == False):
```
Comment (shreyas-9601) [inline order_management.py]: add validations

File: order_management.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            return jsonify({"message":"Order creation failed due to invalid age"}), 400
+        if(orders.user.is_valid_zipcode() == False):
+            return jsonify({"message":"Order creation failed due to invalid zipcode"}), 400
+        if(orders.user.first_monday_born() == False):
+            return jsonify({"message":"Order creation failed due to invalid day of birthday"}), 400
+    return jsonify({"message":"Order creation failed"}), 400
+
+@app.route('/order/<order_id>', methods=['DELETE'])
+def delete_order(order_id):
+    order = OrdersModel.objects(id=str(order_id)).first()
```
Comment (shreyas-9601) [inline order_management.py]: use get instead of first

File: order_management.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    if order:
+        order.is_deleted = True  
+        order.deletion_time = datetime.utcnow()  
+        order.save()
```
Comment (shreyas-9601) [inline order_management.py]: use update instead of save

File: order_services.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+from datetime import datetime
+
+class OrderServices:
+    def save_orders(order_data):
+        order_model = OrdersModel(name=order_data['Name'],birthday=order_data['Birthday'], 
+                                  email=order_data['Email'], state=order_data['State'], zipcode=order_data['ZipCode'])
+        order_model= order_model.save()
+        return order_model
+    
+    def delete_orders(order_id):
```
Comment (shreyas-9601) [inline order_services.py]: use update method

**Review by shubham-zinrelo — APPROVED** [review]

**Pull request merged**